coeus-impl/src/main/resources/META-INF/kc-config-defaults.xml

```diff
@@ -221,5 +221,5 @@
 	<param name="connection.pool.impl">XAPool</param>
 	 
 	<param name="rice.portal.allowed.regex" override="false">^(${application.url}|${rice.server.url})(/.*|)</param>
-	
+	<param name="kc.config.verifier.hard.error">true</param>
 </config>
```

Comment:
oh yeah, you are right. Looks OK. Will wait to fix env build scripts tomorrow to merge.